src/System.IO.Pipelines/StreamExtensions.cs

```diff
@@ -29,21 +29,7 @@ public static IPipeWriter AsPipelineWriter(this Stream stream)
         public static IPipeWriter AsPipelineWriter(this Stream stream, IBufferPool pool)
         {
             var pipe = new Pipe(pool);
-            pipe.CopyToAsync(stream).ContinueWith((task, state) =>
-            {
-                var innerPipe = (Pipe)state;
-
-                if (task.IsFaulted)
-                {
-                    innerPipe.Reader.Complete(task.Exception.InnerException);
-                }
-                else
-                {
-                    innerPipe.Reader.Complete();
-                }
-            }, 
-            pipe, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
-
+            pipe.CopyToAsync(stream);
```

Comment:
This CopyToAsync() could throw, right? For example if Stream.WriteAsync() throws, I think we should observe it. I also think it makes sense to still complete the reader since the intention was to read the entire pipe anyway.